### DISCUSSION TITLE
About project in future ready talent

### DISCUSSION THREAD

**Discussion by chinn-i** [github/india#1054] [GitHub Platform]
Am getting trouble while submitting my project. It is showing that your azure skills are in validation once they are approved then you can submit the project. How many days it takes to validate? Did any one got this?

**Comment by FRTMentor** [comment]
Within 2 - 3 working days will get approved or reject.
If it get any delay more than 3 working days or needed on basis of please reach out the support team.
support@futurereadytalent.in

**Comment by FRTSupport** [comment]
We have unfrozen the submit button without getting the skills validated for now. We urge you to go ahead and submit your project as this is a limited period offer. 

But however, we recommend you to please keep your LinkedIn skill section updated with "Microsoft Azure"
